src/DocumentationBrowserViewExtension/DocumentationBrowserViewExtension.cs

```diff
@@ -436,6 +436,8 @@ protected virtual void Dispose(bool disposing)
             {
                 this.pmExtension.PackageLoader.PackgeLoaded -= OnPackageLoaded;
             }
+
+            PackageDocumentationManager.Instance.MessageLogged -= OnMessageLogged;
```

Comment:
do you need to the 
`       if  (PackageDocumentationManager.Instance.MessageLogged != null)`
like the in other cases?

Reply:
I would usually say yes, but Instance already checks for null internally as it's a singleton.